src/App.tsx

```diff
@@ -15,71 +16,90 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+type State = {
+  selectedGud: string,
+};
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+export class App extends React.Component<{}, State> {
+  state = {
+    selectedGud: 'Jam',
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+  render() {
+    const { selectedGud } = this.state;
 
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+    return (
+      <main className="section container">
+        <h1 className="title is-flex is-align-items-center">
+          {selectedGud ? `${selectedGud} is selected` : 'No goods selected'}
+          {(selectedGud === '')
+            ? null
+            : (
+              // eslint-disable-next-line jsx-a11y/control-has-associated-label
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={() => {
+                  this.setState({
+                    selectedGud: '',
+                  });
+                }}
+              />
+            )}
+        </h1>
```

Comment:
```suggestion
        {selectedGud
            ? (
              <h1 className="title is-flex is-align-items-center">
                {`${selectedGud} is selected`}

                {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
                <button
                  data-cy="ClearButton"
                  type="button"
                  className="delete ml-3"
                  onClick={() => {
                    this.setState({ selectedGud: '' });
                  }}
                />
              </h1>
            )
            : <h1 className="title">No goods selected</h1>
        }
```
you can do it more readable:)